### DISCUSSION TITLE
Hi, how did you make this?

### DISCUSSION THREAD

**Discussion by HGT-123** [CatalystDevOrg/Catalyst#317] [Q&A]
Asking, how on earth did you make a browser?

(i have attempted many times)

**Comment by jdev082** [comment]
@HGT-123 
You can find the methods used for each component by opening the respective file.

Main Interface: **index.html**
Tabs: **tabs.js**

etc

**Reply by HGT-123** [reply]
How did you make the installer, also can i fork this repo to make my own browser??

**Reply by jdev082** [reply]
@HGT-123 Any and all Catalyst source code may be forked into a new browser as long as the fork fits the following requirements within the next 48 hours:

- Issues tab enabled
- Fully rebranded with **zero** references to the Catalyst browser

**Reply by HGT-123** [reply]
Ok, but how id you make the installer

**Reply by jdev082** [reply]
@HGT-123 [Electron Forge](https://www.electronforge.io/).
